supervisor/host/configuration.py

```diff
@@ -192,7 +192,10 @@ def _map_nm_wifi(inet: NetworkInterface) -> WifiConfig | None:
 
         # Signal
         if inet.wireless:
-            signal = inet.wireless.active.strength
+            try:
+                signal = inet.wireless.active.strength
+            except AttributeError:
+                signal = None
```

Comment:
It seems the `active` object is not present in your case. Instead of catching the error, we can just check if that is present:

```suggestion
        if inet.wireless and inet.wireless.active:
            signal = inet.wireless.active.strength
```

Reply:
Thanks, that's a good suggestion. I've added that change and updated the git message to suit.